README.md

```diff
@@ -1,2 +1,98 @@
-# gitlab
+# maubot-gitlab
```

Comment:
Most of the plugin READMEs have just the plugin name, so I think I'll keep it like that here too

```suggestion
# gitlab
```

Reply:
Fixed